tycho-integration-test/src/traces.rs

```diff
@@ -0,0 +1,356 @@
+//! Transaction trace analysis with foundry signature decoding.
```

Comment:
You need not review this file. It's exactly the same as in the testing SDK. Could we import it from somewhere shared? It sucks that it's two different repos.

Reply:
maybe we could have it all here and import from here in the sdk? 🤔